config/configuration.sample.php

```diff
@@ -4,17 +4,27 @@
 class config
 {
 	/* General */
-	public $sitename = "Leantime"; //Name of your site, can be changed later
+	public $sitename = "CALAW"; //Name of your site, can be changed later
```

Comment:
Could you remove the personal settings in this line and line 10+11

Reply:
removed the personal config  commit 268b5fe9992bd4edc9b47bf9fbe48d6da79d7fc0